apps/src/templates/instructions/HelpTabContents.jsx

```diff
@@ -0,0 +1,68 @@
+import {showVideoDialog} from "../../code-studio/videos";
+import React, {Component, PropTypes} from 'react';
+
+const styles = {
+  referenceArea: {
+    marginTop: 20,
+  },
+  videoLink: {
+    display: 'inline-block',
+    margin: 8,
+    fontWeight: 'bold',
+    fontSize: 16,
+    lineHeight: 25 + 'px',
+  },
+  videoThumbnail: {
+    borderRadius: 5,
+    height: 40,
+    width: 'auto',
+    marginRight: 8
+  }
+};
+
+export default class HelpTabContents extends Component {
+  static propTypes = {
+    videoData: PropTypes.shape({
+      src: PropTypes.string,
+      name: PropTypes.string,
+      key: PropTypes.string,
+      download: PropTypes.string,
+      thumbnail: PropTypes.string,
+      enable_fallback: PropTypes.bool,
+      autoplay: PropTypes.bool,
```

Comment:
When we have videoData, are any/all of these fields required? If so we can/should mark them as such.